lib/generators/maintenance_tasks/task_generator.rb

```diff
@@ -12,17 +12,26 @@ class TaskGenerator < Rails::Generators::NamedBase
     class_option :csv, type: :boolean, default: false,
       desc: "Generate a CSV Task."
 
+    class_option :no_collection, type: :boolean, default: false,
```

Comment:
To me, no_collection doesn't really imply a single run. Maybe `once`, `single_run` or `one_off` would make more sense?

Reply:
I mean keeping
```rb
def collection(_task)
  :no_collection
end
```
and `when :no_collection` but renaming the class option, builder class, and class macro method to `single_run`. `process_once` or `one_off` seem fine to me as well, no strong opinion on which is best.